src/app/layout.tsx

```diff
@@ -16,7 +17,10 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
```

Comment:
Moving the header and footer into this layout file. 
All pages created hereafter will automatically have a header and footer rendered as opposed to manually importing headers into each page